docs/rst/user_manual/layout.rst

```diff
@@ -262,3 +268,36 @@ This Dialog will appear every time a new Chartbox is created, or adding a new se
 
 For further details on how to configure a new series refer to :ref:`historic_series` for historic data or
 :ref:`dynamic_series` for dynamic data.
+
+.. _domain_graph:
+
+Domain View
+-----------
+This view in the main panel shows the connections between DataWriters and DataReaders that belong to the same
+DDS Domain.
+They are represented encapsulated inside their physical entities (see :ref:`entities` relationship), and with a
+connection to the topic they are published and subscribed, respectively.
+
+.. figure:: /rst/figures/screenshots/domain_graph.png
+    :align: center
+
+Clicking on any entity displays its detailed information in the :ref:`info_panel`.
+Right click allows changing the alias of the entity, filter the problems to display only the selected entity's problems
+and, in the case of topics, filter the domain graph by topic, so they are displayed only the entities related to the
+selected topic.
```

Comment:
```suggestion
Right click allows changing the alias of the entity, filter the problems to display only the selected entity's problems
and, in the case of topics, filtering the domain graph by topic so that only the entities related to the selected topic are displayed.
```